proprietary/Components/src/utrecht/data-list.tokens.json

```diff
@@ -0,0 +1,60 @@
+{
+  "utrecht": {
+    "data-list": {
+      "margin-block-end": {
+        "value": "{denhaag.space.inline.xs}"
+      },
+      "margin-block-start": {
+        "value": "{denhaag.space.inline.xs}"
+      },
+      "item-key": {
+        "color": {
+          "value": "{denhaag.color.grey.4}"
+        },
+        "font-size": {},
+        "font-weight": {
+          "value": "bold"
+        },
+        "line-height": {
+          "value": "1.5"
+        }
+      },
+      "item-value": {
+        "color": {
+          "value": "{denhaag.color.grey.4}"
+        },
+        "font-size": {},
+        "line-height": {
+          "value": "1.5"
+        }
+      },
+      "rows": {
+        "item": {
+          "margin-block-start": {
+            "value": "{denhaag.space.inline.md}"
+          },
+          "padding-inline-start": {
+            "value": "{denhaag.space.inline.xs}"
+          }
+        },
+        "item-value": {
+          "margin-block-start": {
+            "value": "0"
+          }
+        },
+        "grid-template-columns": {
+          "value": "repeat(auto-fill, clamp(200px, 30%, 100%))"
```

Comment:
I'd wish we could solve this with some more descriptive design tokens with simple `<length>` values, instead of this CSS only construct.

`min-inline-size`/`max-inline-size` come to mind, but not sure how to name them considering they would apply to multiple elements inside this component.

Reply:
I agree. We should implement this with more explicit token values. The system tokens are explicit (little to no implicit values), so we do that here as well.  I'll take another look.

Reply:
I'm looking at `inline-size` (as the target) and `min-inline-size` for the min clamp size. Implicitly `auto-fill` is related to `100%`, so no change is needed there.

Reply:
I'll also update the PR on the Utrecht repo.

Reply:
It turns out I can't do this:
`grid-template-columns: repeat(auto-fill, clamp(var(--utrecht-data-list-rows-min-inline-size), var(--utrecht-data-list-rows-inline-size), 100%));`
But I can do this.
`grid-template-columns: repeat(auto-fill, clamp(20ch, var(--utrecht-data-list-rows-column-width), 100%));`
We don't need the min value, that's now set to a generic width `20-ch` and inline-size is now `column-width`